core-web/libs/template-builder/src/lib/components/template-builder/template-builder.component.stories.ts

```diff
@@ -1,14 +1,174 @@
 import { moduleMetadata, Story, Meta } from '@storybook/angular';
 
+import { NgFor, AsyncPipe } from '@angular/common';
+
+import { DotLayoutBody } from '@dotcms/dotcms-models';
+
+import { TemplateBuilderRowComponent } from './components/template-builder-row/template-builder-row.component';
 import { DotTemplateBuilderStore } from './store/template-builder.store';
 import { TemplateBuilderComponent } from './template-builder.component';
+const data: DotLayoutBody = {
```

Comment:
Should we move this to another file in case we need in case we need to reuse it? @zJaaal